css-brochure-form/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div id="h1"><h1>LEARN MORE</h1></div>
+
+    <form action="#">
+      <label for="companyEmail">COMPANY EMAIL</label><br />
+      <input type="text" placeholder="sally@company.co" id="companyEmail" />
+      <br />
```

Comment:
You should not be using `<br />` for spacing purposes. It's super bad practice. If you need to add space between elements, you should be utilizing `margin`